src/main/java/seedu/ultron/model/ModelManager.java

```diff
@@ -125,7 +128,23 @@ public ObservableList<Opening> getFilteredOpeningList() {
     @Override
     public void updateFilteredOpeningList(Predicate<Opening> predicate) {
         requireNonNull(predicate);
-        filteredOpenings.setPredicate(predicate);
+        filteredOpenings.clear();
```

Comment:
Does the setPredicate method not work with ContainsKeydatePredicate?

Reply:
I changed filteredOpenings to an Observable List instead of FilteredList